File: src/targets/php/guzzle/fixtures/full.php
<?php
require_once('vendor/autoload.php');

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'https://httpbin.org/anything?foo=bar&foo=baz&baz=abc&key=value', [
  'form_params' => [
    'foo' => 'bar'
  ],
  'headers' => [
    'accept' => 'application/json',
    'content-type' => 'application/x-www-form-urlencoded',
    'cookie' => 'foo=bar; bar=baz',
  ],
]);

echo $response->getBody();

File: src/targets/rust/reqwest/fixtures/full.rs
use serde_json::json;
use reqwest;

#[tokio::main]
pub async fn main() {
    let url = "https://httpbin.org/anything";

    let querystring = [
        ("foo", "bar"),
        ("foo", "baz"),
        ("baz", "abc"),
        ("key", "value"),
    ];

    let payload = json!({"foo": "bar"});

    let client = reqwest::Client::new();
    let response = client.post(url)
        .query(&querystring)
        .header("cookie", "foo=bar; bar=baz")
        .header("accept", "application/json")
        .header("content-type", "application/x-www-form-urlencoded")
        .form(&payload)
        .send()
        .await;

    let results = response.unwrap()
        .json::<serde_json::Value>()
        .await
        .unwrap();

    println!("{}", results);
}
